test/Circuit.spec.js

```diff
@@ -191,6 +191,24 @@ describe('Circuit Class', () => {
     brake._circuitOpen = true;
     return circuit.exec('test').then(null, err => {
       expect(err).to.be.instanceof(CircuitBrokenError);
+      expect(err.message).to.equal('Circuit has been opened - The percentage of failed requests (NaN%) is greater than the threshold specified (50%)');
```

Comment:
You can fix the `NaN%` message by making sure the brakes instance as registered at least one call `brake._failureHandler(100)`.